build.sbt

```diff
@@ -56,5 +64,8 @@ lazy val benchmarks =
         "dev.profunktor"    %% "redis4cats-effects" % "0.10.2",
         "io.chrisdavenport" %% "rediculous"         % "0.0.5",
         "io.laserdisc"      %% "laserdisc-fs2"      % "0.4.0"
+      ),
+      scalacOptions in Compile := Seq(
```

Comment:
Is this one necessary?

Reply:
Yes, it's necessary.
```shell
[info] Running scalafix on 10 Scala sources
[error] (benchmarks / Compile / scalafix) scalafix.sbt.InvalidArgument: The Scala compiler option "-Ywarn-unused" is required to use RemoveUnused.
[error] To fix this problem, update your build to use at least one Scala compiler
[error] option like -Ywarn-unused, -Xlint:unused (2.12.2 or above), or -Wunused (2.13 only)
[error] Total time: 10 s, completed 20-Aug-2020, 2:00:11 pm
```
As mentioned in the below document, it needs to have that scala compiler option.
https://scalacenter.github.io/scalafix/docs/rules/RemoveUnused.html#installation

Reply:
I see, thanks!